README.md

```diff
@@ -93,6 +93,31 @@ Mutable data is an anti-pattern. Use the `const const const` keyword to make a c
 const const const pi = 3.14!
 ```
 
+## Variable variable variables
+
+Variable variable variables are the opposite constant constant constants. Instead of never changing their value, variable variable variables **always** change it.
```

Comment:
```suggestion
Variable variable variables are the opposite to constant constant constants. Instead of never changing their value, variable variable variables **always** change it.
```

Reply:
Shouldn't it be "of"?

Reply:
I'm actually not sure about that as I'm not a native English speaker.